dashboards/.lint

```diff
@@ -0,0 +1,15 @@
+exclusions:
```

Comment:
I enabled strict linting so that we will actually exit with an error if the linting fails (before it would never exit with nonzero status). However, I needed to set some exclusions in order to have the dashboard pass despite being valid.